src/org.hdfgroup.hdfview/hdf/view/dialog/NewCompoundAttributeDialog.java

```diff
@@ -78,11 +78,13 @@ public class NewCompoundAttributeDialog extends NewDataObjectDialog {
         "unsigned short (16-bit)", // 4
         "unsigned int (32-bit)",   // 5
         "long (64-bit)",           // 6
-        "float",                   // 7
-        "double",                  // 8
+        "float (32-bit)",          // 7
+        "double (64-bit)",         // 8
         "string",                  // 9
         "enum",                    // 10
-        "unsigned long (64-bit)"   // 11
+        "unsigned long (64-bit)",  // 11
+        "float16 (16-bit)",        // 12
+        "long double (128-bit)"    // 13
```

Comment:
Does the order here affect the order they're displayed in the Dialog box somewhere? If so and if the ordering here isn't set in stone already, it'd be nice to have similar types grouped together.

Reply:
Unfortunately the order here uses the value of the order elsewhere.